app/src/main/java/explorer/city/com/cityexplorer/view/adapter/SearchAdapter.kt

```diff
@@ -26,7 +26,10 @@ class SearchAdapter(listener: OnItemClickListener<String>) : RecyclerView.Adapte
 
     override fun onBindViewHolder(vh: SearchViewHolder, position: Int) {
         val city = results[position]
-        vh.setSearchResult(city)
+        val imageUrl = city.embeddedSearch.cityInfo.embeddedUrbanArea?.urbanArea?.embeddedImage?.photos?.photos?.get(0)?.image?.web
```

Comment:
This is a very deep call into the core structure of an object. One of these objects higher up `city` or `cityInfo` needs a function called `getUrbanImage` or something.

These objects should be abstracting away their inner form from calling classes.